backend/core/database.py

```diff
@@ -666,49 +661,78 @@ def build_radix_tree(self):
                             set(this_translated_asn_list)
                         )
                     rule["neighbors"] = list(rule_translated_neighbor_set)
-
                     conf_obj = {
                         "origin_asns": rule["origin_asns"],
                         "neighbors": rule["neighbors"],
                     }
-                    node.data["confs"].append(conf_obj)
+                    for prefix in rule["prefixes"]:
+                        for translated_prefix in translate_rfc2622(prefix):
+                            ip_version = get_ip_version(translated_prefix)
+                            if self.prefix_tree[ip_version].has_key(translated_prefix):
+                                node = self.prefix_tree[ip_version][translated_prefix]
+                            else:
+                                node = {
+                                    "prefix": translated_prefix,
+                                    "data": {"confs": []},
+                                }
+                                self.prefix_tree[ip_version].insert(
+                                    translated_prefix, node
+                                )
+                            node["data"]["confs"].append(conf_obj)
+                            raw_prefix_count += 1
```

Comment:
the +1 and also the mod might be adding delay on the operations (if we want to be picky :P)

Reply:
Removed %. Will keep +1 since we need the processing count for the logs (informational for the user).